README.oracle_fdw

```diff
@@ -9,33 +9,34 @@ oracle_fdw was written by Laurenz Albe <laurenz.albe@wien.gv.at>
 
 This README contains the following sections:
 
-1. Cookbook
-2. Objects created by the extension
-3. Options
-4. Usage
-5. Installation Requirements
-6. Installation
-7. Internals
-8. Problems
-9. Support
+1. [Cookbook](#1-cookbook)
+2. [Objects created by the extension](#2-objects-created-by-the-extension)
+3. [Options](#3-options)
+4. [Usage](#4-usage)
+5. [Installation Requirements](#5-installation-requirements)
+6. [Installation](#6-installation)
+7. [Internals](#7-internals)
+8. [Problems](#8-problems)
+9. [Support](#9-support)
```

Comment:
I tried various methods but I couldn't ditch the paragraph tag. To solve the consistency issue I added a line between each numbered row (not optimal but that's all I could do)